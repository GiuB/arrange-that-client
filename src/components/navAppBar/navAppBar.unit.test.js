import React from 'react';
import Enzyme, {shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json'

import NavAppBar from './navAppBar';

Enzyme.configure({adapter: new Adapter()});

it('NavAppBar is consistent', () => {
    // Render a NavAppBar
    const navAppBar = shallow(<NavAppBar />);

    expect(toJson(navAppBar, {
        mode: 'deep'
    })).toMatchSnapshot();
});