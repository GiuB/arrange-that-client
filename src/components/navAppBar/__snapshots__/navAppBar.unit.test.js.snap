// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavAppBar is consistent 1`] = `
<Route>
  <Component />
</Route>
`;
